### ISSUE TITLE
Follow Mouse?

### ISSUE THREAD

**Issue by MyTDT-Mysoft** [rdp/screen-capture-recorder-to-video-windows-free#151]
oh well, i came to this because ffmpeg does not have FollowMouse option with gdigrab, and then i found this project, that works ok, despite the configure GUI being slow, instead of a simple native window (why?)

but either way... adding a way to follow mouse would be helpful :)

**Comment by rdp** [comment]
It works ok without follow mouse option?  Or you wish it were added to
both? :)

On Sunday, May 16, 2021, MyTDT-Mysoft ***@***.***> wrote:
> oh well, i came to this because ffmpeg does not have FollowMouse option
with gdigrab, and then i found this project, that works ok, despite the
configure GUI being slow, instead of a simple native window (why?)
>
> but either way... adding a way to follow mouse would be helpful :)
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub, or unsubscribe.<
https://ci6.googleusercontent.com/proxy/XttSQSVmjqqY4EukfgQSUOo2jx-h9sM5Uoi-vFWA6b52DEtUjYq2rLID95CaksYr3ylZUn3cRNAIAjd2liHaNTcKKIwO1TKpAu4BLkwBZBmmqWREpaq5yiARRfs8MOKs2i_rb4zFU18in6mYA_Pa07DNqxyzTG6GhE5s_VRgkHVAHSY9T63iDdp8-9YF3FHSw6GtIyBdsslQD7jiSMbidyMRkwg=s0-d-e1-ft#https://github.com/notifications/beacon/AAADBUDITVNUFMQYSA2CAH3TN7HLZA5CNFSM4462YDTKYY3PNVWWK3TUL52HS4DFUVEXG43VMWVGG33NNVSW45C7NFSM4NJVG53Q.gif>

**Comment by MyTDT-Mysoft** [comment]
yeah it works ok even witout mouse option, but i wish there was one :)
